extensions/wikia/AdEngine/js/spec/lookup/prebid/adapters/appnexusPlacements.spec.js

```diff
@@ -0,0 +1,102 @@
+describe('ext.wikia.adEngine.lookup.prebid.adapters.appnexusPlacements', function () {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This function has 101 lines, which is greater than the 100 lines authorized. Split it into smaller functions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS138)